gnomad_constraint/plots/plotting_functions.R

```diff
@@ -373,3 +374,117 @@ plot_projected_sample_size <- function(df) {
 
   return(list(lof = lof_projections, mis = missense_projections))
 }
+
+plot_decile_change <- function(df) {
+  # Plot of the change in LOEUF deciles between gnomAD versions
+  # df: Dataframe consisting of LOEUF deciles, with the v2 values defined by
+  # 'oe_lof_upper_bin' and the v4 values defined by 'lof.oe_ci.upper_bin_decile'
+  # Returns: ggplot object of the LOEUF decile change from v2 to v4
+
+  # Calculate the decile change by subtracting the v4 value from the v2 value
+  df <- df %>% mutate(
+    decile_change = .data$oe_lof_upper_bin - .data$lof.oe_ci.upper_bin_decile
+  )
+  # Plot decile changes
+  p <- ggplot(
+    df,
+    aes(x = .data$decile_change)
+  ) +
+    geom_histogram(stat = "count") +
+    theme_classic() +
+    theme(
+      axis.title = element_text(colour = "black", size = 25),
```

Comment:
These plots have much larger axes labels and number than the other plots. I'm assuming that is because they are going on poster or in slides. Maybe we should have 2 sets of sizes and an option for using presentation version vs other. Can pull the formatting out into it's own function that get's applied, sort of like we have here for bokeh plots https://github.com/broadinstitute/gnomad_methods/blob/97ec47e45e5a0e5368dad4bbd9fa073204ca3632/gnomad/utils/plotting.py#L327